microprofile/WFLY-19866-promote-mp-rest-client-to-default.adoc

```diff
@@ -0,0 +1,135 @@
+---
+categories:
+- microprofile
+- jaxrs
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/640
+feature-team:
+ developer: jamezp
+ sme:
+  - jamezp
+ outside-perspective:
+  - TBD
+promotes: https://issues.redhat.com/browse/WFLY-19866
+promoted-by: https://github.com/wildfly/wildfly-proposals/issues/640
+---
+= Promote MicroProfile REST Client 4.0 from Preview to Default Stability Level
+:author:            James R. Perkins
+:email:             jperkins@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This document is about promoting the http://download.eclipse.org/microprofile/microprofile-rest-client-4.0/microprofile-rest-client-spec-4.0.html[MicroProfile REST Client 4.0]
+from a stability level of `preview` to a stability level of `default`.
+
+As you'll see in the release notes below, not much was added to the MicroProfile REST Client 4.0 API. There are only two
+new methods added to the `RestClientBuilder` API.
+
+=== User Stories
+
+A user wants to use the MicroProfile REST Client version 4.0 or use MicroProfile Platform 7.0.
```

Comment:
Can you please elaborate how users can benefit from new features brought by new REST Client?